### DISCUSSION TITLE
Add Piper TTS to Text gen WebUI.

### DISCUSSION THREAD

**Discussion by FemBoxbrawl** [oobabooga/text-generation-webui#4893] [Ideas]
Piper TTS is quite realistic, but also arguably the fastest AI text to speech there is. I head it can also run on rasberry pi.

It would be awesome if you could add this, thanks!

**Comment by TheLounger** [comment]
There is an unofficial Piper extension (Windows only): [tijo95/win_tts_piper](https://github.com/tijo95/win_tts_piper).

**Reply by FemBoxbrawl** [reply]
Thanks a lot!

**Reply by Martin-Laclaustra** [reply]
The extension also works for linux. Look in the issues, where I wrote the installation instructions for linux (only requires 1 more line).

**Reply by TheLounger** [reply]
Or use [my fork](https://github.com/TheLounger/piper_tts) which is a WIP but already fixes/improves some things.

**Reply by Martin-Laclaustra** [reply]
@TheLounger I certainly think that you did a great work cleaning @tijo95 's work. If you don't mind, I think it should be appropriate to keep working together in his repo. With your permission I will add your improvements there.

**Reply by tijo95** [reply]
Thank you for taking over the project, @LaLounger & @TheLounger I've given you access to the repository to carry your modifications. make yourself happy.

**Reply by Martin-Laclaustra** [reply]
The repository has been renamed. Currently it is: https://github.com/tijo95/piper_tts
